data/schema/schema_v1.json

```diff
@@ -459,7 +467,8 @@
                               "$ref": "#/definitions/id"
                             },
                             "parent_answer_id": {
-                              "type": "string"
+                              "type": "string",
+                              "description": "The id of an answer which is the parent of one or more child answers, for example a radio option leading to a text field answer."
```

Comment:
At the moment a parent answer can only have one child.

Reply:
updated